src/Apps/Artwork/Utils/createCollectUrl.tsx

```diff
@@ -0,0 +1,65 @@
+import * as qs from "qs"
+import { data as sd } from "sharify"
+
+export const createCollectUrl = ({
+  dimension,
+  category,
+  artistId,
+}: {
+  dimension: "SMALL" | "MEDIUM" | "LARGE" | null
+  category: FilterCategory
+  artistId: string
+}) => {
+  let dimensionRange
+
+  if (dimension === "SMALL") {
+    dimensionRange = "0-1600"
+  } else if (dimension === "MEDIUM") {
+    dimensionRange = "1600-4900"
+  } else if (dimension === "LARGE") {
+    dimensionRange = "4900-*"
+  } else {
+    dimensionRange = "*-*"
+  }
+
+  const query = qs.stringify({
+    page: 1,
+    sort: "-decayed_merch",
+    acquireable: "true",
+    offerable: "true",
+    inquireable_only: "true",
+    dimension_range: dimensionRange,
+    artist_id: artistId,
+  })
+
+  const path = [sd.APP_URL, "collect", filterCategories[category]]
+    .filter(Boolean)
+    .join("/")
+
+  return `${path}?${query}`
+}
+
+export type FilterCategory = keyof typeof filterCategories
+
+// these come from MediumFilter.tsx
+const filterCategories = {
```

Comment:
It looks like this is using the outdated mapping that I fixed in https://github.com/artsy/reaction/pull/2313-- mind updating to match?

Reply:
Will do!